query-engine/driver-adapters/js/adapter-pg/src/pg.ts

```diff
@@ -124,6 +124,12 @@ export class PrismaPg extends PgQueryable<StdClient> implements DriverAdapter {
     return ok(new PgTransaction(connection, options))
   }
 
+  async setDefaultSchema(schema: string): Promise<Result<void>> {
+    const query = { sql: `SET search_path TO ${schema},public`, args: [] }
```

Comment:
This needs some quotes around the schema name to ensure we're escaping it or else this will break. See a test here that gives you some example of schema names that'd need being quoted https://github.com/prisma/prisma-engines/blob/9cc3db6a9d43fdaf3572d95fff2f8015c04688f5/quaint/src/connector/postgres.rs#L1164-L1212

Reply:
And some additional docs [cockroachlabs.com/docs/stable/keywords-and-identifiers#rules-for-identifiers](https://www.cockroachlabs.com/docs/stable/keywords-and-identifiers#rules-for-identifiers) (it's the crdb docs because they are clearer than the pg's one, but the rules are the same. If you really want pg's docs, they're here https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS)

Reply:
Also, there might be ways to set it at connection time https://github.com/brianc/node-postgres/issues/1123 instead of doing it in a separate query, similar to how we've done it with tokio-postgres